File: server/api/mutations/register_container.py
import graphene
from container.models import Container
from api.objects import ContainerType

class RegisterContainerMutation(graphene.Mutation):
    class Arguments:
        container_id = graphene.String()
        tracker_id = graphene.String()
    ok = graphene.Boolean()
    container = graphene.Field(lambda: ContainerType)

    def mutate(self, info, container_id, tracker_id):
        container = Container(container_id=container_id, tracker_id=tracker_id)
        container.save()
        return RegisterContainerMutation(ok=True, container=container)

File: dashboard/src/owner-admin/mutations/registerContainer.js
import gql from 'graphql-tag'

export default gql`
mutation registerContainer($containerId: String!, $trackerId: String!) {
    registerContainer(containerId: $containerId, trackerId: $trackerId) {
      ok
      container {
        trackerId
        containerId
      }
    }
  }`